compiler-rt/test/builtins/Unit/extendhfxf2_test.c

```diff
@@ -55,11 +54,16 @@ int main() {
                        (long double)-__builtin_huge_valf64x()))
```

Comment:
```suggestion
                       makeNegativeInf80()))
```

This function is currently missing from fp_test.h, but you can add it below makeInf80().

Reply:
same with negativeInf16() by the way.

Reply:
ok.